bio.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en" style="background-color:rgb(48, 48, 48); color: rgb(226, 225, 223);">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Bio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>Biography</h1>
+        <p>Junior Engineer</p>
+
+    </header>
+    <nav class="menu">
+        <div>
```

Comment:
Nav is clear and specific as to each anchor.  To increase accessibility, I'd place each a tag in the nav as an unordered list ul, and have each item as an li (list item).  This aides a screen reader in understanding the construction of a navbar.  Happy to show you if you'd like.